db/migrate/20211112035233_add_looking_for_work_to_developers.rb

```diff
@@ -0,0 +1,14 @@
+class AddLookingForWorkToDevelopers < ActiveRecord::Migration[7.0]
+  def up
+    add_column :developers, :looking_for_work, :boolean
+
+    Developer.where("available_on < ?", Date.today + 1.year).update_all(looking_for_work: true)
```

Comment:
Do we have any plans to communicate with all the registered profiles about this change? or it's implicit

Reply:
I was planning on it being implicit, but would love to discuss more in the Discussion forum #86.